src/main/java/ToDo.java

```diff
@@ -0,0 +1,11 @@
+public class ToDo extends Task{
+
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
```

Comment:
It is good to specify overridden methods

Reply:
Thank you for the comment.

Reply:
Thank you for the kind comment.